embc-app/Migrations/20190506213507_EvacueeRegistration-Evacuee-EvacueeAddress-Tables.cs

```diff
@@ -92,7 +92,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         column: x => x.CommunityId,
                         principalTable: "Communities",
                         principalColumn: "Id",
-                        onDelete: ReferentialAction.Cascade);
+                        onDelete: ReferentialAction.Restrict);
```

Comment:
restrict can be dangerous, maybe DeleteBehavior.ClientSetNull?

Reply:
Yes that would be safer.  I'll change it in 'Develop' and push it with the next Master merge.